src/pages/admin/event/[eventId]/[year]/stats/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Attendee } from "@/components/RegistrationTable/columns";
+import { BasicInformation } from "@/types";
+import { GetServerSideProps } from "next";
+import { useRouter } from "next/router";
+
+type Props = {
+    initialData: BasicInformation[] | null
+}
+
+export default function Statistics() {
+    const router = useRouter()
+
+    return (
+        <main className="bg-primary-color min-h-screen">
+          <div className="container mx-auto p-10 flex flex-col">
+                <span>
+                    <h2 className="text-white">Event Statistics</h2>
+                    <p className="text-baby-blue font-poppins">Statistics {">"} {router.query.eventId} {router.query.year}</p>
+                    {/* 
+                    insert charts here
+                    */}
+                </span>
+            </div>
+        </main>
+    );
+}
+
+
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    const { eventId, year } = context.params as { eventId: string, year: string };
+
+    try {
+        const data = await fetchRegistationData(eventId, year);
+        return { props: { initialData: data } };
+    } catch (error) {
+        console.error("Failed to fetch initial data:", error);
+        return { props: { initialData: null } };
+    }
+}
+
+async function fetchRegistationData(eventId: string, year: string) {
```

Comment:
Typo in `fetchRegistationData` -> `fetchRegistrationData`

Not a big concern for now since it's just mock data, but let's make sure this `fetchRegistrationData` method is shared in a future `utils` class for `RegistrationTable` to also use